pmg/templates/_create_alert.html

```diff
@@ -23,15 +33,33 @@ <h2 class="search-q">"{{ q }}"</h2>
         <p>
           <a class="btn btn-info" type="button" data-q="{{ q }}" data-type="{{ filters.type }}" data-committee="{{ filters.committee }}">Create alert</a>
         </p>
-          Get <a href="{{ url_for('email_alerts') }}">free email alerts</a> when we add new information that match these search terms.
+          Get free email alerts when we add new information that match these search terms.
       </div>
 
       <div class="email-alert-box remove-alert {% if not saved_search %} hidden {% endif %}">
-        <a class="btn btn-warning" type="button" data-id="{{ saved_search.id }}">Remove alert</a>
-      </div>
+        <p>
+          <a class="btn btn-warning" type="button" data-id="{{ saved_search.id }}">Remove alert</a>
+        </p>
+          You are subscribed to <b>"{{ q }}"</b>
```

Comment:
This feels wordy, since the details of the search are just to the left of this box in any case.

Perhaps "We'll send you an email when we find new items for this search." or similar?